content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md

```diff
@@ -1,3 +1,11 @@
+---
+html: connect-your-rippled-to-the-xrp-test-net.html
+funnel: Build
+doc_type: Tutorials
+category: Manage the rippled Server
```

Comment:
is Server supposed to be capitalized? It's not below

Reply:
This looks weird, but it's correct. It's in title case, but "rippled" is in lowercase per the official stylization.